samples/analyze.v1.js

```diff
@@ -35,28 +35,24 @@ function analyzeSentimentOfText(text) {
   };
 
   // Detects the sentiment of the document
-  client
-    .analyzeSentiment({document: document})
-    .then(results => {
-      const sentiment = results[0].documentSentiment;
-      console.log(`Document sentiment:`);
-      console.log(`  Score: ${sentiment.score}`);
-      console.log(`  Magnitude: ${sentiment.magnitude}`);
-
-      const sentences = results[0].sentences;
-      sentences.forEach(sentence => {
-        console.log(`Sentence: ${sentence.text.content}`);
-        console.log(`  Score: ${sentence.sentiment.score}`);
-        console.log(`  Magnitude: ${sentence.sentiment.magnitude}`);
-      });
-    })
-    .catch(err => {
-      console.error('ERROR:', err);
-    });
+  const [result] = await client.analyzeSentiment({document: document});
```

Comment:
You can just shorten this to `{ document }`

Reply:
updated as suggested Thanks